app/lib/bing_api.rb

```diff
@@ -0,0 +1,54 @@
+require 'net/https'
+require 'uri'
+require 'json'
+require 'pry'
+
+class BingApiV7
+  def initialize
+    @accessKey = Rails.application.secrets.bing_api_v7
```

Comment:
BAD BAD naming for variable

Reply:
just took it from example. what is better, api_key?

Reply:
`bing_access_key` for example
or better to make it as nested key, e.g.:
```
bing:
  access_key: BING_ACCESS_KEY_GOES_HERE
```

and within application
```
Rails.application.secrets.bing['access'_key']
```